doc/release/README.multilocale.rst

```diff
@@ -0,0 +1,233 @@
+============================
+Multilocale Chapel Execution
+============================
+
+To run Chapel on multiple locales, the following steps are required:
+
+
+0. The following information describes a fairly generic use of
```

Comment:
Oh wait, am I being dumb? Does auto-numbering require you to specify the starting point? If so, nevermind my comment